package/caddy/caddyFile

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Let's use nginx for this, no need to introduce yet another component to do the same thing. I can provide nginx configuration for this.

Reply:
Agreed. We'll remove caddy as part of this PR, and rely on you guys adding in nginx to the image